src/keybinds.h

```diff
@@ -10,15 +10,32 @@ double elapsedF2, elapsedF12, elapsedReloadCfg;
 uint64_t last_f2_press, last_f12_press, reload_cfg_press;
 
 #ifdef HAVE_X11
-bool key_is_pressed(KeySym ks) {
+bool keys_are_pressed(std::vector<KeySym>& keys) {
 
     if (!init_x11())
         return false;
 
     char keys_return[32];
+    std::vector<bool> pressed;
```

Comment:
Might as well be size_t if just use for counting

Reply:
Replaced with `size_t`